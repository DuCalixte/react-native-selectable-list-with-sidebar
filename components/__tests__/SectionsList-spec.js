'use strict';

const React = require('React');
const ReactTestRenderer = require('react-test-renderer');

const SectionList = require('SectionList');

describe('SectionList', () => {
  it('renders empty list', () => {
    const component = ReactTestRenderer.create(
      <SectionList sections={[]} renderItem={({item}) => <item v={item.key}/>}/>,);
    expect(component).toMatchSnapshot();
  });
});
