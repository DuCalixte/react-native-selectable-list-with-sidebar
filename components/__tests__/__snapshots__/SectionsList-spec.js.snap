// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionList renders empty list 1`] = `
<RCTScrollView
  ItemSeparatorComponent={undefined}
  data={Array []}
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  onViewableItemsChanged={undefined}
  renderItem={[Function]}
  scrollEventThrottle={50}
  sections={Array []}
  stickyHeaderIndices={Array []}
  stickySectionHeadersEnabled={true}
  updateCellsBatchingPeriod={50}
  windowSize={21}
>
  <View />
</RCTScrollView>
`;
